### PR TITLE
[vrfmgrd]: Support VNET setting

### PR DESCRIPTION
**Pull request by marian-pritsak**
Signed-off-by: Marian Pritsak <marianp@mellanox.com>

### CODE REVIEW

File: cfgmgr/vrfmgr.cpp [hunk 1]
```diff
@@ -137,6 +140,41 @@ bool VrfMgr::setLink(const string& vrfName)
...
+        else
+        {
+            vnetValues.push_back(fv);
+        }
+    }
+
+    tunnelValues.emplace_back("vrf", vrfName);
+
+    m_appVnetTableProducer.set(vrfName, vnetValues);
+    m_appTunnelMapTableProducer.set(tunnelName, tunnelValues);
```
Comment (prsunny) [inline cfgmgr/vrfmgr.cpp]: There is a tunnel map name as well to uniquely identify the the mapping. Can you check if its set?

```
VXLAN_TUNNEL_MAP:{{tunnel_name}}:{{**tunnel_map**}}
    "vni": {{ vni_id}}
    "vrf": {{ vrf_name }}
```
You can use the save Vnet/Vrf name for tunnel_map name as well.

File: cfgmgr/vrfmgr.cpp [hunk 2]
```diff
@@ -137,6 +140,41 @@ bool VrfMgr::setLink(const string& vrfName)
...
+
+    for (const auto& fv : values)
+    {
+        const auto& name = fvField(fv);
+
+        if (name == "vni")
+        {
+            tunnelValues.push_back(fv);
+        }
+        else if (name == "tunnel")
```
Comment (prsunny) [inline cfgmgr/vrfmgr.cpp]: "vxlan_tunnel" - This is the name in schema.

File: cfgmgr/vrfmgr.cpp (hunk 1 above)
Comment (marian-pritsak) [reply cfgmgr/vrfmgr.cpp]: Done

File: cfgmgr/vrfmgr.cpp (hunk 2 above)
Comment (marian-pritsak) [reply cfgmgr/vrfmgr.cpp]: Yes, I was looking at a diagram, thanks
Done

File: cfgmgr/vrfmgr.cpp [hunk 3]
```diff
@@ -137,6 +140,41 @@ bool VrfMgr::setLink(const string& vrfName)
...
+        {
+            tunnelName = fvField(fv);
```
[the remark this answers is not in the document]
Comment (marian-pritsak) [reply cfgmgr/vrfmgr.cpp]: Done

**Comment by lguohan** [comment]
@marian-pritsak , which vstest covers this change?

**Pull request merged**